lib/rules/nested-interactive.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "nested-interactive",
```

Comment:
This looks like it can run on virtual nodes only. Can you put in some tests for that?